packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -172,50 +184,54 @@ const Messages: FunctionComponent<MessagesProps> = ({
   }, [messageLayoutNotifications])
 
   useEffect(() => {
-    const hideInfoPopupsTimeout = setTimeout(() => {
-      messagePopupNotifications
-        .filter((item) => item.method === NotificationMethod.Popup)
-        .forEach((item) => {
-          removeNotification(item.id)
-        })
+    messagePopupNotifications.length > 0 &&
+      removePopupNotification(messagePopupNotifications)
+    threadPopupNotifications.length > 0 &&
+      removePopupNotification(threadPopupNotifications)
+    // AUTO DISABLED - fix me if you like :)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Not sure if this should have `AUTO DISABLED - fix me if you like :)` - it seems to be added by purpose and should stay.

Reply:
Removed in [cdb4aea](https://github.com/mudita/mudita-center/pull/996/commits/cdb4aead3df31d3ade0b8b629cc868077614298c)